### PR TITLE
Added steps to disable csvcopy to diable

### PR DESCRIPTION
**Pull request by tarukumar**
Added steps to disable csv copy in cluster
Signed-off-by: Tarun Kumar <takumar@redhat.com>

### CODE REVIEW

**Comment by tarukumar** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1041/console

File: tests/Tests/700__sandbox/701__performance_sandbox.robot [hunk 1]
```diff
@@ -79,3 +79,8 @@ Verify Sandbox Toolchain Data
                   ...   expected in toolchain result=> ${km_data[0]} : ${m_value}
            END
     END
+
+Performance Suite Setup
+    #Oc Apply    kind=OLMConfig    src=${CURDIR}/olm.yaml   #Not working
```
Comment (jgarciao) [inline tests/Tests/700__sandbox/701__performance_sandbox.robot:84]: [Documentation] missing(explaining that  we are disabling CopiedCSVs and why)

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/700__sandbox/701__performance_sandbox.robot (hunk 1 above)
Comment (tarukumar) [reply tests/Tests/700__sandbox/701__performance_sandbox.robot]: Done

**Review by jgarciao — DISMISSED** [review]

**Review by mwaykole — DISMISSED** [review]

**Review by mwaykole — APPROVED** [review]

**Review by pablofelix — APPROVED** [review]
lgtm

**Pull request merged**